bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -0,0 +1,79 @@
+  <p>Once building the new platform was underway, Siddoway, who’d been managing the community on his own and routing feedback and comments to employees as needed, began recruiting product managers to join him. Fembro and his colleague, Ania Safko, both desktop product managers, were the first to participate when Connect launched quietly in the spring. After a few weeks, Siddoway met with them to discuss how things were going.</p>
+  <p>Siddoway was pleased by what he heard. The PMs reported that a lot of engaging, interesting conversations were happening. Fembro’s initial feedback request had generated dozens of responses about bookmarks and tabs, topics that had elicited strong opinions among many in the community.</p>
+  <p>As with any launch, there were rough spots to be worked out. Siddoway initially thought they’d have two product managers commit to two week cycles of being on the platform. But was that ideal? The team decided it was still too early to tell as much would depend on the number of comments coming in and how much time PMs would be spending on the task.</p>
+  <p>The conversation progressed, with Siddoway and the managers agreeing that the community’s engagement would be most valuable to their work at two key stages of product development. At the start, when they’re brainstorming ideas for features and products, and at the end, when the developers are testing what they’ve built.</p>
+  <p>“It’s really useful for us to engage with users in exploratory situations where we know we want to improve a certain experience but we haven’t made a decision about the exact solution,” Safko said. “We are interested in hearing how people use something or how they think about the underlying need so we can better understand them.”</p>
+  <p>The goal isn’t to define specific solutions, but to let the ideas fly. “We’re asking the community to come up with the most innovative and blue-sky creative features that they can possibly think of that address a problem we’re trying to solve,” said Fembro. “Sometimes you'll see a really thorough post that clearly identifies a scenario or opportunity gap, and outlines a possible solution for us to consider.”</p>
+  <p>Once a feature has been incorporated into the browser, the Connect community can offer valuable feedback on whether it is performing as intended “in the wild,” Fembro said.</p>
+  <p>The team needed to discuss another, less tangible issue: how could they keep conversations from getting derailed? “Mozilla has built a rich culture over the years and from the beginning has been people-focused,” said Fembro. “A lot of our loyal users are very entrenched into our development process, which comes along with a lot of opinions.”</p>
+  <div class="c-stories-article-float-container left">
+    {{ stories_image(
+      image_url="img/stories/placeholder.png",
+      alt="",
```

Comment:
**issue (accessibility):** once we get final images, it would be nice to have some descriptive alt text